templates/content/node--oe-sc-event--full.html.twig

```diff
@@ -45,10 +45,14 @@
 {% endset %}
 
 <article{{attributes}}>
-  {{ pattern('inpage_navigation', {
-    title: 'Page content',
-    links: inpage_navigation_links,
-    content: inpage_navigation_fields,
-    full_layout: true,
-  }) }}
+  {% if inpage_navigation_links is null %}
```

Comment:
Please use is empty, so we also skip if empty array.

Reply:
(resolve once read, ignore)
Btw the variable should really be named `inpage_navigation_content` instead of `inpage_navigation_links`, but out of scope here.